controller/tests/controller_server/csi_controller_server_test.py

```diff
@@ -93,6 +93,13 @@ def _test_request_with_wrong_secrets(self, storage_agent):
 
         self.request.secrets = []
 
+    def _test_request_already_processing(self, storage_agent, request_attribute, object_id):
+        storage_agent.side_effect = self.storage_agent
+        self.servicer.sync_lock.ids_in_use(request_attribute, object_id)
+        response = self.get_tested_method()(self.request, self.context)
+        self.assertEqual(self.context.code, grpc.StatusCode.ABORTED)
+        self.assertEqual(response, self.get_tested_method_response_class()())

```

Comment:
what does it mean that the responses are "equal"? I assume it's not a "deep" comparison.
did you mean to compare their type?

Reply:
is it more clear now?

Reply:
why not just `type(response), self.get_tested_method_response_class()`?

Reply:
ok, done